### DISCUSSION TITLE
Ability to send env and build-env parameters

### DISCUSSION THREAD

**Discussion by svdbrg** [vercel/community#4590] [Help]
We're looking to migrate our pipelines from using the Vercel CLI to this plugin. We are however heavily dependent on supplying custom environment variables to our Vercel builds and runtimes, which seems to be a current limitation.

Any plans on adding this feature?

**Comment by amyegan** [comment]
Hey @svdbrg! The good news is that's already available when deploying via the CLI. The [`--env`](https://vercel.com/docs/cli/deploy#env) option can be used to provide environment variables at run time. And the [`--build-env`](https://vercel.com/docs/cli/deploy#build-env) option can be used to provide variables to the build step.

https://vercel.com/docs/cli/deploy#unique-options

**Reply by svdbrg** [reply]
Hey @amyegan, thanks for your reply. I'm aware about the CLI options, we're doing that already. We are however looking to move away from CLI deployments and use this plugin instead, but maybe the purpose of this plugin is for basic use only and not the "advanced" setup we're looking for?
